src/violations.rs

```diff
@@ -4253,6 +4253,20 @@ impl AlwaysAutofixableViolation for EndsInPeriod {
     }
 }
 
+define_violation!(
+    pub struct UseImperativeMood(pub String);
```

Comment:
I think this violation should rather be named NonImperativeMood, see 5bf6da0db7ef76fd81f2932eac4a18d99e380789 for the reasoning.

Reply:
I would kind of love that the name of the violation is also in imperative mood 😁

Reply:
Well the idea is that you can have `Allow <violation-name>` which is imperative mood :)

Reply:
Will change to `ImperativeMood`, c.f. `FirstLineCapitalized`, `EndsInPunctuation`.

Reply:
The violation structs currently aren't named consistently so referencing two others doesn't really make sense. I think the `Allow <violation-name>` convention from Rust makes sense which would mean we'd have to always name the negative case ... which actually does makes sense as well for violations since "imperative mood" isn't a violation but what's expected .... "non-imperative mood" would be the violation.